src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -719,6 +719,56 @@ public static function get_nodes_to_translate() {
 					),
 				),
 			),
+			 'archive-posts' => array(
+                'conditions' => array( self::TYPE => 'archive-posts' ),
```

Comment:
@amElnagdy, could you please check your indentation?

Reply:
sorry, should be ok now.